src/index.ts

```diff
@@ -35,22 +35,23 @@ function keyDownHandler(event: KeyboardEvent) {
 
   currentTriePosition = newTriePosition
   if (newTriePosition instanceof Leaf) {
+    const target = event.target as HTMLElement
+    const scopedElements = newTriePosition.children.filter(element => element.hasAttribute('data-hotkey-scope'))
+    const nonScopedElements = newTriePosition.children.filter(element => !element.hasAttribute('data-hotkey-scope'))
```

Comment:
if you accept https://github.com/github/hotkey/pull/51#discussion_r719425409 then these won't be needed any more.

```suggestion
```